prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.ts

```diff
@@ -169,6 +200,17 @@ export class CareSettingComponent extends BaseEnrolmentProfilePage implements On
     if (!this.careSettings?.length) {
       this.addCareSetting();
     }
+
+    // Initialize Health Authority form even if it might not be used:
+    // Create unchecked checkboxes for each known Health Authority.
+    this.form.controls.selectableHealthAuthorities = this.fb.array(this.configService.healthAuthorities.map(() => this.fb.control(false)));
```

Comment:
We shouldn't be adding controls directly to the controls array of a form.  There is an API for adding controls if needed, but it feels like this might be possible with just the single form group - `enrolleeHealthAuthorities`.  Currently, you're dynamically adding a new `FormArray` and populating it with controls, but could it be accomplished using `enrolleeHealthAuthorities` to avoid transferring between the `FormArray`s in the JSON method?

Reply:
@mtpultz About this feedback and feedback re: comments, let's talk later.  Thanks.